crates/turbopack-core/src/version.rs

```diff
@@ -186,17 +185,20 @@ pub enum Update {
 #[derive(PartialEq, Eq, Debug, Clone, TraceRawVcs, ValueDebugFormat, Serialize, Deserialize)]
 pub struct TotalUpdate {
     /// The version this update will bring the object to.
-    pub to: VersionVc,
+    #[turbo_tasks(trace_ignore)]
+    pub to: TraitRef<VersionVc>,
```

Comment:
Why is this necessary?

Reply:
`compute_update_stream` is only executed the `UpdateStreamItem` changes (since it only this is awaited). But `VersionVc` might be equal while the underlying content can change. So when trying to send 2 `TotalUpdate` with `to` pointing to the same Vc (but with different content) it won't send that.